BUILD

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Google LLC
```

Comment:
Rename this file (and all other BUILD files) to BUILD.bazel, and similarly for the WORKSPACE file.
This is the recommended convention outside of google, since it makes the purpose of these files easy to research for non-Bazel experts.

Reply:
Done